pyquil/quil.py

```diff
@@ -643,7 +650,15 @@ def address_qubits(program, qubit_mapping=None):
             result.append(Gate(instr.name, instr.params, remapped_qubits))
         elif isinstance(instr, Measurement):
             result.append(Measurement(qubit_mapping[instr.qubit], instr.classical_reg))
-
+        elif isinstance(instr, Pragma):
+            new_args = []
+            for arg in instr.args:
+                if isinstance(arg, QubitPlaceholder):
```

Comment:
add a comment here describing what you said in the thread (sometimes args are actually things that need to be preserved)